common/runtime/client.go

```diff
@@ -12,7 +12,10 @@ import (
 	"github.com/kyma-project/kyma-environment-broker/common/pagination"
 )
 
-const defaultPageSize = 100
+const (
+	defaultPageSize = 100
+	RuntimesLimit   = 1000
```

Comment:
So we leave it hard-wired, no configuration possible?

Reply:
I think this parameter will not change in the future, so I would leave it hard-wired. Do you think it should be configurable?